src/rpm/builder.rs

```diff
@@ -67,17 +69,31 @@ async fn async_file_mode(_file: &async_std::fs::File) -> Result<u32, RPMError> {
     Ok(0)
 }
 
+/// Returns a value that can be used for associating several package builds as being part of one operation
+///
+/// You can use any value, but the standard format is "${build_host} ${build_time}"
+///
+/// See `RPMBuilder::cookie()`
+pub fn get_build_cookie() -> String {
+    let build_time = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .unwrap()
```

Comment:
Is this an infallible operation? Might be nice to be `expect()` if so.

Reply:
It ought to be effectively infalliable since it only returns `Err` when "earlier is later than self".  So the system timestamp would have to be less than 0.